### ISSUE TITLE
Resource trees loose keyboard focus when selecting node

### ISSUE THREAD

**Issue by faubulous** [faubulous/mentor-vscode#33]
When selecting a node the definitions tree looses keyboard focus and one cannot navigate it using the keyboard.
